### ISSUE TITLE
Add `stop on yellow` parameter for GitHub action

### ISSUE THREAD

**Issue by mzwiesler** [mercedes-benz/sechub#2602]
The `stop on yellow` is a possible configuration of the Sechub scan as described [here](https://mercedes-benz.github.io/sechub/latest/sechub-client.html#configuration-overview). However, it is not possible to define it using the Github Action. It would be great if this would be included in the Github Action.

**Comment by Abdullah-Benomar-Shahen** [comment]
@de-jcup I suppose the action located at _github-actions/scan/action.yml_ need to be extended with the mentioned flag as described in the docs, right? If yes, I can work on the extension.

**Comment by mzwiesler** [comment]
> @de-jcup I suppose the action located at _github-actions/scan/action.yml_ need to be extended with the mentioned flag as described in the docs, right? If yes, I can work on the extension.

Exactly. This is what I had in mind. Would be great if you could take it up.

**Comment by thomgun** [comment]
@Abdullah-Schahin ,
are there any progress with this ?

**Comment by Abdullah-Schahin** [comment]
@thomgun will push it this week. This went of the radar somehow! Thanks for the reminder.
